packages/shared/src/components/auth/ForgotPasswordForm.tsx

```diff
@@ -20,7 +21,22 @@ function ForgotPasswordForm({
   onClose,
 }: ForgotPasswordFormProps): ReactElement {
   const [emailSent, setEmailSent] = useState(false);
-  const onSendEmail = () => setEmailSent(true);
+  const [timer, setTimer] = useState(0);
```

Comment:
I think we can create a custom hook here to abstract these conditions. Something like `useTimer` - I think we can utilize it in some areas in our auth system.

Reply:
Good point, definitely something we can use, i'll refactor